service/build.gradle

```diff
@@ -75,12 +75,18 @@ tasks.register("pactTests", Test) {
 	useJUnitPlatform {
 		includeTags "pact-test"
 	}
+	testLogging {
+		showStandardStreams = true
+	}
```

Comment:
This provides more information about what version is being executed:

![image](https://github.com/DataBiosphere/terra-billing-profile-manager/assets/484484/b6bacbea-dd82-47ed-82cd-c1a6c26244c5)